staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/deepcopy.go

```diff
@@ -268,5 +268,21 @@ func (in *JSONSchemaProps) DeepCopy() *JSONSchemaProps {
 		}
 	}
 
+	if in.XListMapKeys != nil {
```

Comment:
(did not find a good place to raise question about this commit in whole; put at the fist line) understood this commit was cherry-picked from upstream. Is this commit absolutely needed for openapi/SMD version bumping, or just putting it in for convenience?

Reply:
This is part of kube-openapi upgrade. more detail can be found in community PR: https://github.com/kubernetes/kubernetes/pull/77354

Reply:
ok; go for it.